crates/cdk-signatory/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    #[cfg(feature = "grpc")]
```

Comment:
```suggestion
    println!("cargo:rerun-if-changed=src/proto/signatory.proto");
    
    #[cfg(feature = "grpc")]
```

This will make sure `cargo` only recompiles when the `proto` file changes.